erdos/ray/ray_cluster.py

```diff
@@ -0,0 +1,55 @@
+import json
+import re
+
+from erdos.cluster import Cluster
+
+
+class RayCluster(Cluster):
+    def __init__(self, redis_address="", can_install_ray=False):
+        super(RayCluster, self).__init__()
+        self.redis_address = redis_address
+        self.can_install_ray = can_install_ray
+
+    def __del__(self):
+        self.broadcast_command("ray stop")
+        super(RayCluster, self).__del__()
+
+    def install_ray(self, node):
```

Comment:
This is great, but we don't have to worry about installing the dependency on the workers in the v0. We can assume that we have all the ROS/Ray dependencies on each node.